public/Day 02/style.css

```
@@ -1,682 +1,145 @@
-        #scrollToTopBtn:hover {
-        background-color: #0056b3;
-        transform: scale(1.1);
-        }
-
-    
\ No newline at end of file
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Poppins" (font-family-name-quotes)

(font-family-name-quotes)